src/main/java/org/radarbase/appserver/exception/InvalidFileDetailsException.java

```diff
@@ -0,0 +1,13 @@
+package org.radarbase.appserver.exception;
+
+public class InvalidFileDetailsException extends IllegalArgumentException {
```

Comment:
Can you map these to HTTP codes using `@ResponseStatus(HttpStatus.XYZ)`? Perhaps Expectation failed?

Reply:
Added! Can you check this please. I have never used this mechanism.

Reply:
yes looks good, thanks